arch/arm64/core/xen/Kconfig

```diff
@@ -25,3 +25,11 @@ config XEN_DOM0LESS
 	help
 	  Configures Zephyr as DomU, that can be started on Dom0less
 	  setup.
+
+config XEN_INTERFACE_VERSION
+	hex "Xen interface version"
+	default 0x00040e00
```

Comment:
I'm not sure that we need to add default value, but for sure not 4.13 (all changes in mainline and here were targeted to 4.17)